src/bugsnag/ErrorPrompt.tsx

```diff
@@ -0,0 +1,123 @@
+// NOTE: do not include any other files here, try to keep this component separate from the rest
+
+import React from "react";
+
+import "./ErrorPrompt.scss";
+
+const CHUNK_NOT_IN_CACHE = /Loading chunk [\d]+ failed/;
+
+const LOAD_ERROR_TITLE = "Loading Error";
+const LOAD_ERROR_MESSAGE = `
+Apologies for the interruption.
+There seems to be a problem loading necessary file for the proper functioning of this app.
+
+This can happen if there has been an update.
+Please try reloading the page to get the newer version.
+
+In case the problem persists, please contact \u2728 Sparkle and provide the error details.
+`;
+
+const UNEXPECTED_ERROR_TITLE = "Unexpected Error";
+const UNEXPECTED_ERROR_MESSAGE = `
+You can try clearing the error in hope the app will keep working as expected
+or if that doesn't help - reload the page.
+
+In case the problem persists, please contact \u2728 Sparkle and provide the error details.
+`;
+
+interface ChunkLoadError extends Error {
+  type?: string;
+  request?: string;
+}
+
+export interface FallbackComponentProps {
+  error: Error | ChunkLoadError;
+  info: React.ErrorInfo;
+  clearError: () => void;
+}
+
+export const isChunkLoadError = (error?: unknown): error is ChunkLoadError =>
+  error instanceof Error && error.name === "ChunkLoadError";
+
+export const ErrorPrompt: React.FC<FallbackComponentProps> = ({
+  error,
+  info,
+  clearError,
+}) => {
+  const chunkLoadError = isChunkLoadError(error) ? error : undefined;
```

Comment:
There isn't object of type `ChunkLoadError` since react/webpack produces regular `new Error` and attaches `name`, `type` and `request` values to it. The `isChunkLoadError` type guard helps narrow the type.